tests/Fixtures/TestBundle/Entity/Container.php

```diff
@@ -33,7 +34,7 @@ class Container
     private $id;
 
     /**
-     * @ApiProperty(subresource=true)
+     * @ApiProperty(subresource=@ApiSubresource())
```

Comment:
Should it be a `PROPERTY` annotation? I don't think we need it, and there is no value in adding metadata factories for subresources IMO. Lmk.